Resources/doc/first_steps.rst

```diff
@@ -0,0 +1,456 @@
+First steps using the ODM
+=========================
+
+The best way to understand the Doctrine MongoDB ODM is to see it in action.
+In this section, you'll walk through each step needed to start persisting
+documents to and from MongoDB.
+
+.. sidebar:: Code along with the example
+
+    If you want to follow along with the example in this chapter, create
+    an ``AcmeStoreBundle`` via:
+
+    .. code-block:: bash
+
+        php bin/console generate:bundle --namespace=Acme/StoreBundle

```

Comment:
We should probably rewrite this entire chapter to be bundle-less.

Reply:
This is not outdated, I just messed up line separators. Will be done later